proto/src/rangeserver.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package rangeserver;
+
+message RangeId {
+    string keyspace_id = 1;
+    string range_id = 2;
+}
+
+service RangeServer {
+    // Our PreFetch rpc accepts a PreFetchRequest and returns HelloReplies
+    rpc PreFetch (PreFetchRequest) returns (HelloReply);
+}
+
+message PreFetchRequest {
+    // Request message contains the range and they key to prefetch
+    repeated RangeId range = 1;
```

Comment:
I assume the purpose of repeated here is to support prefetching multiple keys. If so, I think it's better to create a type that captures a range,key tuple and then have that be repeated, instead of having two independently repeated fields.

Reply:
Updated